server/preprocessing/other-scripts/text_similarity.R

```diff
@@ -53,14 +52,25 @@ if(!is.null(params_file) && !is.na(params_file)) {
   params <- NULL
 }
 
+if ('language' %in% params){
+    language = params$language
```

Comment:
Here you have to take into account that the frontend may send "all" for all languages. In this case, my suggestion would be to use also set the language to `'english'`.